README.md

```diff
@@ -239,6 +239,10 @@ Legend:
 
 ## Known Issues
 
+- The crashpad backend currently has no support for notifying the crashing
+  process, and can thus not properly terminate sessions or call the registered
+  `before_send` hook. It will also lose any events that have been queued for
+  sending at time of crash.
 - Attachments are currently in _Preview_ and may not be available to your
   organization. Please see [Event Attachments] for more information.
```

Comment:
We can remove the attachments disclaimer now. Also, what if we rename this section to "Known Limitations"?